ruby-training/trainees/alan/test_katas/02/fizzbuzz/fizzbuzz_game.rb

```diff
@@ -1,31 +1,31 @@
 class FizzBuzzGame
     
-    def self.it_is_fizz? (number)
+    def self.is_it_fizz? (number)
         number % 3 == 0
     end
 
-    def self.it_is_buzz? (number)
+    def self.is_it_buzz? (number)
         number % 5 == 0
     end
 
-    def self.it_is_fizzbuzz? (number)
-        it_is_buzz?(number) and it_is_fizz?(number)
+    def self.is_it_fizzbuzz? (number)
+        is_it_buzz?(number) and is_it_fizz?(number)
     end
 
-    def self.it_has_a_3? (number)
+    def self.has_it_a_3? (number)
```

Comment:
Check the grammar here please, it doesn't read correctly. Maybe something like `contains_a_three`. Also, no digits in variable names please, if you must write a number, use the english word for it ("three" instead of "3").